README.md

```diff
@@ -101,20 +101,14 @@ If you want to have the newest (unreleased) `dask-sql` version or if you plan to
 
 Create a new conda environment and install the development environment:
 
-    conda env create -f continuous_integration/environment-3.9-jdk11-dev.yaml
+    conda env create -f continuous_integration/environment-3.9-dev.yaml
 
 It is not recommended to use `pip` instead of `conda` for the environment setup.
-If you however need to, make sure to have Java (jdk >= 8) and maven installed and correctly setup before continuing.
-Have a look into `environment-3.9-jdk11-dev.yaml` for the rest of the development environment.
 
 After that, you can install the package in development mode
 
     pip install -e ".[dev]"
 
-To recompile the Java classes after changes have been made to the source contained in `planner/`, run
-
-    python setup.py build_ext
-
```

Comment:
No recompilation needs to happen on changes to the Rust code?

Reply:
They would just run another `python ./setup.py install` guess we should add that in the README though.